src/services/stellar/asset.service.ts

```
@@ -0,0 +1,259 @@
+      return await this.transactionService.signAndSubmit(transaction, [sourceKeypair]);
+    } catch (error) {
+      throw new StellarError('Bulk transfer failed', error as Error, {
+        transferCount: transfers.length,
+        source: sourceKeypair.publicKey(),
+      });
+    }
+  }
+
+  validateAmount(amount: string): boolean {
+    const numAmount = parseFloat(amount);
+    return !isNaN(numAmount) && numAmount > 0 && numAmount <= 9223372036854775807;
```

Biome (2.1.2)

[error] This number literal will lose precision at runtime.

The value at runtime will be 9223372036854776000

(lint/correctness/noPrecisionLoss)